recipes/jsoncons/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+from conan import ConanFile
+from conan.tools.files import get, copy
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.50.0"
+
+
+class JsonconsConan(ConanFile):
+    name = "jsoncons"
+    description = "A C++, header-only library for constructing JSON and JSON-like data formats, with JSON Pointer, JSON Patch, JSON Schema, JSONPath, JMESPath, CSV, MessagePack, CBOR, BSON, UBJSON"
+    topics = ("json", "csv", "cpp", "json-serialization", "cbor", "json-parser", "messagepack", "json-pointer", "json-patch", "json-diff", "bson", "ubjson", "json-parsing", "jsonpath", "jmespath", "csv-parser", "csv-reader", "jsonschema", "json-construction", "streaming-json-read")
+    homepage = "https://github.com/danielaparker/jsoncons"
+    url = "https://github.com/danielaparker/jsoncons"
+    author = "Daniel Parker danielaparker@yahoo.com"
+    license = "Boost Software License"
```

Comment:
We use https://spdx.org/licenses/BSL-1.0.html short names to be standardized

```suggestion
    license = "BSL-1.0"
```